### DISCUSSION TITLE
Lesson 3 completed

### DISCUSSION THREAD

**Discussion by yeahChibyke** [Cyfrin/foundry-full-course-cu#376] [Show and tell]
I just finished the third lesson in the youtube series. I learnt about ABIs, imports, inheritance, etc., and I also built smart contracts to further solidify my knowledge of these concepts.
![Lesson_3_completed](https://github.com/Cyfrin/foundry-full-course-f23/assets/56419077/42fa8aeb-d728-47bf-b99a-579597ec4a08)

**Comment by infinite969** [comment]
Nice!
